java-manta-client/src/main/java/com/joyent/manta/client/crypto/AesCtrCipherDetails.java

```diff
@@ -59,42 +61,82 @@ public boolean plaintextSizeCalculationIsAnEstimate() {
 
     @Override
     public long[] translateByteRange(final long startInclusive, final long endInclusive) {
```

Comment:
Let's do that once we integrate. My thoughts as well. I've created #159 to capture that work.